Dockerfile.assisted-service-build

```diff
@@ -2,7 +2,10 @@ FROM golang:1.14.3
 
 ENV GO111MODULE=on
 
-RUN apt-get update && apt-get install -y docker.io libvirt-clients awscli python3-pip postgresql \
+RUN echo 'deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && \
+    curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/Release.key | apt-key add -
+
+RUN apt-get update && apt-get install -y podman libvirt-clients awscli python3-pip postgresql \
```

Comment:
Running podman inside docker can be complicated to setup. Would it be easier to replace what skipper+Dockerfile.assisted-service-build is doing with an additional make target or ansible to setup the environment. And then run podman directly on the host?.